transmit-lib/src/cmr/transmit/echo/tokens.clj

```diff
@@ -48,7 +48,7 @@
 
     (let [[status parsed body] (r/rest-post context "/tokens/get_token_info"
                                             {:headers {"Accept" mt/json
-                                                       "Echo-Token" (transmit-config/echo-system-token)}
+                                                       "Authorization" (transmit-config/echo-system-token)}
                                              :form-params {:id token}})]
```

Comment:
I think this change specifically is going to break CMR when deployed - this is the main call that passes tokens to Legacy-Services, and Legacy-Services I predict will not be expecting an `Authorization` header. I think you should dig around Legacy-Services to find where it is accepting token headers to make sure this will be accepted (or for now, just pass in both headers)